packages/vscode-mdx/src/document-drop-edit-provider.js

```diff
@@ -1,11 +1,30 @@
 /**
+ * @typedef {import('mdast').RootContent} RootContent
  * @typedef {import('vscode').DocumentDropEditProvider} DocumentDropEditProvider
  * @typedef {import('vscode').DataTransferItem} DataTransferItem
  */
 
+import path from 'node:path'
 import {Uri, WorkspaceEdit} from 'vscode'
 import {toMarkdown} from 'mdast-util-to-markdown'
 
+// https://github.com/microsoft/vscode/blob/1.83.1/extensions/markdown-language-features/src/languageFeatures/copyFiles/shared.ts#L29-L41
+const imageExtensions = new Set([
```

Comment:
this const appears to be exported by VSCode, would we want to leverage it directly?
Filtering it by `MediaKind.Image`

Reply:
Are you referring to the fact the constant is exported internally in the linked file? I don’t see any exports in the `markdown-language-features` extension’s `activate` function.

Reply:
> Are you referring to the fact the constant is exported internally in the linked file? 

Yes

Reply:
Then those aren’t exposed anywhere.